draft-ietf-quic-recovery.md

```diff
@@ -1151,7 +1152,8 @@ OnLossDetectionTimeout():
     DetectLostPackets(pn_space)
   // Retransmit crypto data if no packets were lost
   // and there are still crypto packets in flight.
-  else if (crypto packets are in flight):
+  else if (crypto packets are in flight &&
+           crypto data to send):
     // Crypto retransmission timeout.
     RetransmitUnackedCryptoData()
     crypto_count++
```

Comment:
else if (client && (no 1-RTT keys)
    send a 1200B packet
or something to that effect

Reply:
Also in the other PR.